yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```

Comment:
Why this file? And why is it in the root folder?

Reply:
Argh, deleted. Probably ran `yarn start` or something without realizing you had to go inside `/website` and this was generated.